src/allmydata/scripts/tahoe_mv.py

```diff
@@ -47,6 +49,7 @@ def mv(options, mode="move"):
     if path:
         to_url += "/" + escape_path(path)
 
+    from_path = unicode(from_path, "utf-8")
```

Comment:
Similar pattern here as in `tahoe_ls.py` where this value is used once above before it is converted.  All good or just untested?